src/000-SCRIPT_OBJ/ROGUE/RoguePlayer.js

```diff
@@ -208,6 +209,22 @@ App.Rogue.Player = function() {
                 this._lightLevel = 1;
             }
 
+            if (this._lightDuration > 0 && Math.random() > 0.85) {
+                // get approximate direction to exit
+                /** @type {App.Rogue.XY} */
+                const exit = this._level.getExit();
+                const dirToExit = Math.atan2(xy.y - exit.y, exit.x - xy.x); // y axis it from top to bottom
+
+                function dirString(angle) {
+                    const sector = angle / (Math.PI / 4);
+                    if (angle < -3 || sector > 3) return "west";
+                    if (sector > 1) return "north";
+                    if (sector < -1) return "south";
+                    return "east";
+                }
+                App.Rogue.Engine._textBuffer.write(`A light breeze through a wall hole swaied your torch's flame, suggesting the exit is to the ${dirString(dirToExit)}`);
```

Comment:
> A light breeze through a wall hole swaied your torch's flame

"swaied" isn't really an english word, or used like that kind of verb. Better to write something like:

"A light breeze from the ${direction} causes your torch to flicker."

It's okay to leave it kind of ambiguous. People will easily figure it out.

Reply:
Winds in caves are very weak (a few cm/s typically). A torch would not flicker from that, but the flame would just tilt a bit. Also, the wind can blow in either direction, so I'd randomly  add PI to the direction to the exit, OK?

Reply:
I would have been okay with just "you think the stairs down are <direction>", because the point of this is to just offset the increased difficulty that was added by light not going through walls. I definitely don't want a message which could be interpreted as a wrong clue unless someone happens to contemplate the physics of cave drafts??

Reply:
It is now "A light breeze from the ${dirString(dirToExit)} causes your torch to flicker."